client/src/app/api/rest.ts

```diff
@@ -543,6 +545,17 @@ export const removeFileTaskgroup = ({
   );
 };
 
+export const getMigrationWaves = () =>
+  axios.get<MigrationWave[]>(MIGRATION_WAVES).then((response) => response.data);
```

Comment:
It's a detail, the following suggestion returns the same meanwhile the objective is be consistent (and we have a backlog ;) ).  
```
export const getMigrationWaves = (): Promise<MigrationWave[]> =>
  axios.get(MIGRATION_WAVES).then((response) => response.data);
```